frontend/src/static/js/main.js

```diff
@@ -211,6 +215,23 @@ window.app = new window.Vue({
       }
     },
 
+    renderZoomTabHash() {
+      const hash = window.hashParams;
+      const info = {
+        author: hash.tabAuthor,
+        repo: hash.tabRepo,
+        avgCommitSize: hash.avgCommitSize,
+        sinceDate: hash.zoomSince,
+        untilDate: hash.zoomUntil,
```

Comment:
would be better to name the key to be the same as the hash to avoid some misinterpretation.

Reply:
the current work does not support merge ramp yet.

Reply:
then i guess either hold this one or the other. Otherwise we will end up reviewing things that might not get merged.

Reply:
holding the merge group option will be a better choice.

Reply:
Holding on to the merge group option PR will be a better choice.

Reply:
Actually, since the merge group option PR has a higher priority and restoring the merge ramps would be quite complicated, putting this PR on hold first might be a better choice.